dubbo-metadata/dubbo-metadata-definition-protobuf/src/main/java/org/apache/dubbo/metadata/definition/protobuf/ProtobufTypeBuilder.java

```diff
@@ -48,6 +48,8 @@ public class ProtobufTypeBuilder implements TypeBuilder, Prioritized {
      */
     private static Type STRING_LIST_TYPE;
 
+    private final boolean protobufExist;
```

Comment:
`static` may be better

Reply:
In a multi-class loader scenario, static may cause class loading exceptions